airbyte-config/init/src/main/resources/seed/destination_specs.yaml

```diff
@@ -3594,8 +3594,6 @@
                   \ more memory. Allowed values: min=5MB, max=525MB Default: 5MB."
                 type: "integer"
                 default: 5
-                minimum: 5
-                maximum: 525
```

Comment:
I am really confused why the spec changed. I did not touch the `spec.json` at all.